dashboard/app/views/pd/application/teacher_application_mailer/accepted_no_cost_registration.html.haml

```diff
@@ -7,7 +7,7 @@
 %p
   Congratulations! We are excited to offer you a seat in
   = @application.effective_regional_partner_name + "'s"
-  2020-21 Professional Learning Program for
+  2021-22 Professional Learning Program for
```

Comment:
could we use @application.year for these?  since we have [this method](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/pd/application/teacher2021_application.rb#L521)?

Reply:
oh yeah we could! The year will show up as '2021-2022' instead of '2021-22', but I think that would be ok?

Reply:
Updated! Also updated the previews to use the active application year hash (they were hardcoded to 2019). The previews will show 2020-2021 until we merge in this year's teacher applications, so we may want to inform anyone who reviews the previews before then that it will update automatically.

Reply:
Nice! Good call on the previews!